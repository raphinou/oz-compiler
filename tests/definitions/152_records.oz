% constant records
local
   R = rec(f1 b:f2 c:f3 r:rec(1 2 3))
in
   {Show R}
   {Show R.1}
   {Show R.r}
   {Show R.r.1}
end

