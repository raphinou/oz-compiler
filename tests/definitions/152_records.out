rec(1:f1 b:f2 c:f3 r:rec(1 2 3))
f1
rec(1 2 3)
1
